code/modules/vore/eating/vorepanel_vr.dm

```diff
@@ -217,6 +217,14 @@
 		dat += "<br><a href='?src=\ref[src];b_tastes=\ref[selected]'>Can Taste:</a>"
 		dat += " [selected.can_taste ? "Yes" : "No"]"
 
+		//Can belly taste?
```

Comment:
Wrong comments?

Reply:
"belly" is also used for people putting other people in their mouths and holding them in there, especially macro-micro. I have a "belly" with the name and verbs set up purely so I an generate the display text of "Scree is attempting to cram Smal Microsen into his gaping maw!"

Reply:
I mean the "can belly taste", when it's on the options for setting brute/burn damage.
I'm pretty sure it was copypasted from the actual taste control above and the comments were untouched.